robot/stream/octet.robot

```diff
@@ -0,0 +1,81 @@
+*** Settings ***
+Documentation       A test suite for examples/octet_stream/key-value-store 
+
+Library             Process
+Library             String
+Library             OperatingSystem
+
+Test Setup          Start Key Value Store Server
+Test Teardown       Stop Server And Clients
+
+
+*** Variables ***
+${SERVER_HOST}      localhost
+${SERVER_PORT}      55520
+${SERVER_PATH}      /path/to/the/server
+${CLIENT_PATH}      /path/to/the/client
+
+${FIRST_CLIENT_INPUT}         { "type": "put", "key": "A", "value": "26" }\n{ "type": "get", "key": "A" }\n{ "type": "del", "key": "A" }\n{ "type": "get", "key": "A" }     
+${FIRST_CLIENT_BASELINE}      reply: \nreply: 26\nreply: 26\nreply: {"error":"no_such_key"}
+${SECOND_CLIENT_INPUT}        { "type": "put", "key": "B", "value": "27" }\n{ "type": "get", "key": "B" }\n{ "type": "del", "key": "B" }\n{ "type": "get", "key": "B" }      
+${SECOND_CLIENT_BASELINE}     reply: \nreply: 27\nreply: 27\nreply: {"error":"no_such_key"}    
+
+
+*** Test Cases ***
+Send Some message
```

Comment:
That's not a descriptive test case name. Also, there's no indication on what's the point of doing the same thing twice in the test?